### ISSUE TITLE
[Bug]: opentelemetry_operator - Certificate & Issuer kinds can't be recognize

### ISSUE THREAD

**Issue by niv1612** [aws-ia/terraform-aws-eks-blueprints#631]
### Welcome to Amazon EKS Blueprints!

- [X] Yes, I've searched similar issues on [GitHub](https://github.com/aws-ia/terraform-aws-eks-blueprints/issues) and didn't find any.

### Amazon EKS Blueprints Release version

v4.0.8

### What is your environment, configuration and the example used?

Terraform version:  v1.0.11

### What did you do and What did you see instead?

  ```
enable_opentelemetry_operator = true
  opentelemetry_operator_helm_config = {
    version = "0.7.0"
  }

```
Those are the settings that I turned on. 

And this is the error that I got:

`
Error: unable to build kubernetes objects from release manifest: [unable to recognize "": no matches for kind "Certificate" in version "cert-manager.io/v1", unable to recognize "": no matches for kind "Issuer" in version "cert-manager.io/v1"]
│
│   with module.eks_blueprints_kubernetes_addons.module.opentelemetry_operator[0].module.operator.helm_release.addon[0],
│   on .terraform/modules/eks_blueprints_kubernetes_addons/modules/kubernetes-addons/helm-addon/main.tf line 1, in resource "helm_release" "addon":
│    1: resource "helm_release" "addon" {`

**Comment by bonclay7** [comment]
Hey, in which example where you deploying this?

**Comment by niv1612** [comment]
[fargate-serverless](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/examples/fargate-serverless)

**Comment by bonclay7** [comment]
Should be solved by https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/627

**Comment by bryantbiggs** [comment]
closed in #627
